main.py

```diff
@@ -0,0 +1,69 @@
+"""Proxy server to communicate a client to ARTIQ."""
+
+from contextlib import asynccontextmanager
+import json
+import os
```

Comment:
We have written `import` lines first, and then `from ... import` lines.

```suggestion
import json
import os
from contextlib import asynccontextmanager
```

Reply:
I didn't notice it😅 I will rearrange it.